tests/foreman/ui/test_role.py

```diff
@@ -567,30 +567,42 @@ def test_positive_create_overridable_filter(self):
                 menu_locators['menu.hosts'], timeout=3))
 
     @stubbed
-    @tier2
-    def test_positive_update_role_taxonomies(self):
-        """Update role taxonomies which applies only to non-overrided filters
-        in role
+    @tier1
+    def test_negative_update_role_taxonomies_overridden_filters(self):
+        """Overridden role filters taxonomies doesnt updates on update of role
```

Comment:
`Overridden role filters taxonomies doesnt updates on update of role taxonomies` is not readable enough from language perspective